packages/itinerary-body/src/index.js

```diff
@@ -200,7 +204,17 @@ ItineraryBody.propTypes = {
    * - leg: the transit leg
    * - stopsExpanded: whether the intermediate stop display is currently expanded
    */
-  TransitLegSummary: PropTypes.elementType.isRequired
+  TransitLegSummary: PropTypes.elementType.isRequired,
+  /**
+   * A custom icon component inserted into the transit alert toggle button
+   * within a transit leg
+   */
+  AlertToggleIcon: PropTypes.elementType,
+  /**
+   * A custom icon component inserted into the transit alert body component
+   * within a transit leg
+   */
```

Comment:
It might be nice to note that if nothing is provided, a default icon still shows up.

Reply:
amended the docstrings to reflect this ^